build_tools/github_actions/configure_ci.py

```diff
@@ -77,6 +110,16 @@
 RUNNER_ENV_DEFAULT = "prod"
 RUNNER_ENV_OPTIONS = [RUNNER_ENV_DEFAULT, "testing"]
 
+CONTROL_JOBS = frozenset(["setup", "summary"])
+
+POSTSUBMIT_ONLY_JOBS = frozenset(
+    [
+        "build_test_all_windows",
+        "build_test_all_macos_arm64",
+        "build_test_all_macos_x86_64",
+    ]
+)
```

Comment:
(Coming from this [discussion on Discord](https://discord.com/channels/689900678990135345/1080178290188374049/1126585388773945466))

Can we also add parts of a job matrix to this list? For example, `a100` in `test_gpu`: https://github.com/openxla/iree/blob/c1ec61dfd2cb58229058684f96992c18eba913c3/.github/workflows/ci.yml#L274-L289

Reply:
Hmmm that gets trickier because we can't just look at the yaml file to get the job names. That might be a disadvantage of using the job matrix. I could do the matrix expansion myself in this script, but that gets a bit complicated... When we're doing that level of invasive examination of GitHub workflow definitions, I think we should probably just generate the whole thing. Do you mind if I punt on this? I think this is still an improvement as is, even though skipping a100s is an important use case.

Reply:
> Do you mind if I punt on this?

Sure. We can at least split the matrix into two jobs as a quick-and-dirty solution if needed...